src/internal_modules/roc_rtp/validator.cpp

```diff
@@ -66,6 +66,21 @@ bool Validator::check_(const packet::RTP& prev, const packet::RTP& next) const {
         sn_dist = -sn_dist;
     }
 
+    if (!!next.capture_timestamp && !!prev.capture_timestamp) {
+        const bool valid_capt_ts =
+            core::ns_equal(next.capture_timestamp, prev.capture_timestamp,
+                           config_.max_ts_jump)
```

Comment:
This call to ns_equal here looks weird.

Maybe rename ns_equal to ns_in_delta?